public/scripts/controllers/snake.js

```diff
@@ -9,7 +9,7 @@ var app = app || {};
   snake.blocksY = 30;
   snake.foodCount = 4;
   snake.initialSize = 3;
-  snake.debug = false;
+  snake.debug = true;
   snake.initialized = false;
 
   let points = [];
```

Comment:
Just so you know, you can declare all these variables with one `let` separated by commas.
`let board, width, height, cellWidth,....`
It isn't bad practice to declare each separately, it just looks a bit cleaner.